### PR TITLE
chore: adding details to changelog

### PR DESCRIPTION
**Pull request by LPLafontaineB**
### Description

This PR adds more details to changelog entries for the next release.

### Issue Number(s)

None

### Contribution checklist
 - [ ] Tests have been added for boss room and/or utilities pack
 - [x] Release notes have been added to the [project changelog](../CHANGELOG.md) file and/or [package changelog](../Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md) file
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] JIRA ticket ID is in the PR title or at least one commit message
 - [ ] Include the ticket ID number within the body message of the PR to create a hyperlink

### CODE REVIEW

**Review by fernando-cortez — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 ### Removed
 *
 ### Fixed
-* Subscribing to a message channel while unsubscribing is pending (#675)
+* Subscribing to a message channel while unsubscribing is pending (#675). This issue prevented us from subscribing to a message channel after having unsubscribed to it if no message had been sent between the un-subscription and the new subscription.
 * Using ```Visible``` instead of ```Enabled``` to make sure RNSM continues updating when off (#702)
 * Some NetworkBehaviours are disabled instead of being destroyed (#718) - This preserves the index order for NetworkBehaviours between server and clients, resulting in no indexing issue for sending/receiving RPCs.
 * Scene Bootstrapper: future proofing bootstrap scene so we don't rely on Startup's path. MTT-3707. (#735)
```
Comment (SamuelBellomo) [inline CHANGELOG.md:50]: @fernando-cortez any learnings for #742?

Comment (fernando-cortez) [reply CHANGELOG.md:49]: Not really? It was just a pattern to reset triggers that other Actions implemented that wasn't carried over into this Action.

**Review by SamuelBellomo — APPROVED** [review]
Just did Phil's PRs. I believe that was it. I just put it as ready for review

**Review by chrispope — APPROVED** [review]
lgtm

**Pull request merged**